transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -103,4 +105,33 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
       submittedBy = donorInput.read[String]("submitted_by")
     )
   }
+
+  def transformAntibody(antibodyInput: Msg): Antibody = {
+    // Use regular expressions to remove everything but the actual target name, which should be the same across
+    // all targets in the list. Remove "synthetic_tag" type targets.
+    val mappedTarget = antibodyInput
+      .read[Array[String]]("targets")
+      .collectFirst { case EncodeIdPattern(accession) => accession } // filter out non-matches
```

Comment:
Instead of `collectFirst` here, should we collect everything and make sure they all agree? A mismatch would be a case where logging a warning is appropriate.

Reply:
There is now a `CommonTransformations.readId` method which might be useful here